daemon/container.c

```diff
@@ -2062,9 +2061,9 @@ container_destroy(container_t *container)
 		}
 	mem_free0(file_name_created);
 
-	char *path = container_token_paired_file_new(container);
-	unlink(path);
-	mem_free0(path);
+	if (smartcard_release_pairing(container)) {
+		ERROR_ERRNO("Can't remove token paired file!");
```

Comment:
check errno directly after syscalls not after wrapper function.

Reply:
Done